_pages/welcome-to-TTS/getting-started.md

```diff
@@ -237,6 +237,17 @@ In a follow-up E&I engagement we, well, experiment and iterate on the findings f
 
 The short version is that projects are staffed through the [18F Staffing repo](https://github.com/18F/staffing/issues), and you can always comment on an issue to express your interest.
 
+## Our approach to Agile
+
+We are known for our Agile approach to both software development and procurement (sometimes called “modular procurement”). In fact, many of our partners come to us because they want to learn how to “be more Agile.” If you take away nothing else from this page, take this: Agile is not something you *do*. Agile is something you *are*. 
```

Comment:
```suggestion
We are known for our Agile approach to both software development and procurement (sometimes called “modular procurement”). In fact, many of our partners come to us because they want to learn how to “be more Agile.” If you take away nothing else from this page, take this: 

> Agile is not something you *do*. Agile is something you *are*. 
```